File: app/src/main/java/me/yxp/qfun/hook/base/HookItemAnnotation.java
package me.yxp.qfun.hook.base;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

@Target(ElementType.TYPE)
@Retention(RetentionPolicy.RUNTIME)
public @interface HookItemAnnotation {

    String TAG();

    String desc() default "";

    String TargetProcess() default "";

}

File: annotation/src/main/java/me/yxp/qfun/annotation/HookItemAnnotation.kt
package me.yxp.qfun.annotation

@Target(AnnotationTarget.CLASS)
@Retention(AnnotationRetention.RUNTIME)
annotation class HookItemAnnotation(
    val tag: String,
    val desc: String = "",
    val category: String = HookCategory.OTHER,
    val process: String = ""
)